src/displayAllGames.php

```diff
@@ -4,7 +4,7 @@ function displayAllGames(array $games)
 {
     $title = '';
     foreach ($games as $game) {
-        if ($game->deleted !== 1) {
+        if ($game->deleted === 0) {
```

Comment:
Rather than filtering out deleted/undeleted like this, it is actually more efficient to do with and SQL where condition.

Reply:
I might need some help with that one because I have no idea where do do the query